x/tokenomics/keeper/msg_server_update_params_test.go

```diff
@@ -0,0 +1,137 @@
+package keeper_test
+
+import (
+	"testing"
+
+	testkeeper "github.com/pokt-network/poktroll/testutil/keeper"
+	"github.com/pokt-network/poktroll/testutil/sample"
+	"github.com/pokt-network/poktroll/x/tokenomics/keeper"
+	"github.com/pokt-network/poktroll/x/tokenomics/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestUpdateParams_Validity(t *testing.T) {
+	tokenomicsKeeper, ctx := testkeeper.TokenomicsKeeper(t)
+	srv := keeper.NewMsgServerImpl(*tokenomicsKeeper)
+
+	params := types.DefaultParams()
+	tokenomicsKeeper.SetParams(ctx, params)
+
+	tests := []struct {
+		desc string
+
+		req *types.MsgUpdateParams
+
+		expectErr     bool
+		expectedPanic bool
+		expErrMsg     string
+	}{
+		{
+			desc: "invalid authority address",
+
+			req: &types.MsgUpdateParams{
+				Authority: "invalid",
+				Params: types.Params{
+					ComputeUnitsToTokensMultiplier: 1,
+				},
+			},
+
+			expectErr:     true,
+			expectedPanic: false,
+			expErrMsg:     "invalid authority",
+		},
+		{
+			desc: "incorrect authority address",
+
+			req: &types.MsgUpdateParams{
+				Authority: sample.AccAddress(),
+				Params: types.Params{
+					ComputeUnitsToTokensMultiplier: 1,
+				},
+			},
+
+			expectErr:     true,
+			expectedPanic: false,
+			expErrMsg:     "authority address is incorrect",
+		},
+		{
+			desc: "invalid ComputeUnitsToTokensMultiplier",
+
+			req: &types.MsgUpdateParams{
+				Authority: tokenomicsKeeper.GetAuthority(),
+
+				Params: types.Params{
+					ComputeUnitsToTokensMultiplier: 0,
+				},
+			},
+
+			expectErr:     true,
+			expectedPanic: true,
+			expErrMsg:     "invalid compute to tokens multiplier",
+		},
+		{
+			desc: "successful param update",
+
+			req: &types.MsgUpdateParams{
+				Authority: tokenomicsKeeper.GetAuthority(),
+
+				Params: types.Params{
+					ComputeUnitsToTokensMultiplier: 1000000,
```

Comment:
Relieved to see this kind of value. I was going to suggest having a float type of the multiplier to have better/fine grained control over adjusting up or down the rewards.

But having an initial/common value in that order works too (I like to see it as numerator with implicit denominator). It's not really code related but I believe we should document the magnitude order of the values we'll be using. Maybe it's stated somewhere in the design doc and I've missed it.